### ISSUE TITLE
Additional details on 9f170335b8ff703292c43e1c

### ISSUE THREAD

**Issue by tobiasjwm** [keybase/client#2524]
I am receiving an "Items cannot be copied to "_name_" because you don't have permission to read them error. This is _my_ public folder.
![items_cannot_be_copied](https://cloud.githubusercontent.com/assets/5116035/14250528/d8a1a412-fa4c-11e5-9c6a-be62290b1a5a.png)

my log id: 9f170335b8ff703292c43e1c

**Comment by strib** [comment]
According to the logs, there is no user currently logged in.  Usualy when the keybase service starts it pops up a pinentry dialogue for you to enter your Keybase passphrase.  It's possible that dialogue was behind some other windows.  Did you ever see it?

You can try to fix this by running `keybase login`.

**Comment by cjb** [comment]
I see:
```
   "Username": "",
   "UserID": "",
   "Device": null,
   "LoggedIn": false,
   "KeychainUnlocked": false,
   "LocalKeySecurityLoaded": false,
   "SessionIsValid": false,
   "SessionStatus": "no session",
```
So I think this machine's just never been logged into.  Coming later this week we'll pop up a login screen in that circumstance, but for now you need to run `keybase login` on a new machine.

**Comment by tobiasjwm** [comment]
Gentlemen,

Thank you for this. I did not see a login prompt. I had also thought I had provisioned the device, but when I went to login, I found that I was incorrect. Once provisioned, I was able to use my folder as expected.

I guess my confusion lay in having the CLI tool installed and having functions such as mounting another user’s public directory work as expected. I must have been distracted half-way through setup. I am easily distracted.

-- 
Tobias Morrison
Apple Certified Technical Coordinator
GlobalMac IT
440-941-1622
